docs/source/index.rst

```diff
@@ -10,7 +10,11 @@ Welcome to fastplotlib's documentation!
    :maxdepth: 2
```

Comment:
set this as 1, it will prevent this annoying thing lol

![image](https://user-images.githubusercontent.com/9403332/211989659-34e64d13-e7f6-4eea-b14a-f986e31b838c.png)